incubator/artifactory/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  # This uses a "fullname" template (see _helpers)
+  # Basing names on .Release.Name means that the same chart can be installed
+  # multiple times into the same namespace.
+  name: {{template "fullname" .}}
+  labels:
+    # The "heritage" label is used to track which tool deployed a given chart.
+    # It is useful for admins who want to see what releases a particular tool
+    # is responsible for.
+    heritage: {{ .Release.Service | quote }}
+    # This makes it easy to search for all components of a release using kubectl.
+    release: {{ .Release.Name | quote }}
+    # This makes it easy to audit chart usage.
+    chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+spec:
+  replicas: {{default 1 .Values.replicaCount}}
+  template:
+    metadata:
+      labels:
+        app: {{template "fullname" .}}
+        release: {{.Release.Name | quote }}
+    spec:
+      containers:
+      - name: artifactory
+        # Making image configurable is not necessary. Making imageTag configurable
+        # is a nice option for the user. Especially in the strange cases like
+        # artifactory where the base distro is determined by the tag. Using :latest
+        # is frowned upon, using :stable isn't that great either.
+        image: "{{default "docker.bintray.io/jfrog/artifactory-pro" .Values.image}}:{{default "4.13.2" .Values.imageTag}}"
+        imagePullPolicy: {{default "IfNotPresent" .Values.pullPolicy}}
+        resources:
+          requests:
+            cpu: {{default "200m" .Values.Cpu}}
+            memory: {{default "2048M" .Values.Memory}}
+        ports:
+        - containerPort: 8081
+          name: http
+        # This (and the volumes section below) mount the config map as a volume.
+        volumeMounts:
+        - mountPath: /var/opt/jfrog/artifactory/etc
+          name: etc
+        - mountPath: /var/opt/jfrog/artifactory/logs
+          name: logs
+        - mountPath: /var/opt/jfrog/artifactory/data
+          name: data
+      volumes:
+        - name: etc
+        configMap:
+          name: {{ template "fullname" . }}
+        - name: data
+        configMap:
+          name: {{ template "fullname" . }}
+        - name: logs
+        configMap:
+          name: {{ template "fullname" . }}
```

Comment:
You list PVC in your instructions, but you do not define any PVC components in your chat?